Lib/inspect.py

```diff
@@ -1954,7 +1956,13 @@ def _signature_get_partial(wrapped_sig, partial, extra_args=()):
                     new_params[param_name] = param.replace(default=arg_value)
                 else:
                     # was passed as a positional argument
-                    new_params.pop(param.name)
+                    # Do not pop if it is a Placeholder
+                    #   and change kind to positional only
```

Comment:
```suggestion
                    #   but change its kind to positional-only
```

(Check whether we say "positional only" or "positional-only" also; I would have said "positional-only" but now I'm not sure anymore).